sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -336,6 +336,11 @@ class depends_on {
     depends_on(NodeTN... nodes);
 };
 
+class depends_on_all_leaves {
+public:
+  depends_on_all_leaves() = default;
```

Comment:
nitpick, the indentation of  `public:` and the constructor isn't the same as the definitions above